src/components/Login/Login.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import {
+    Anchor,
+    Button,
+    Checkbox,
+    Divider,
+    Flex,
+    Group,
+    Paper,
+    PasswordInput,
+    Stack,
+    Text,
+    TextInput,
+} from '@mantine/core';
+import {GoogleLogin, GoogleOAuthProvider} from '@react-oauth/google';
+import PreloaderExt from "../Preloader/PreloaderExt";
+import {useLogin} from "../../hooks/useLogin";
+import {useStyles} from "./style";
+import {Navigate} from "react-router-dom";
+
+export const Login = () => {
+
+    const {isLoading, type, form, formHandler, onClickHandler, onGoogleSuccess, isAuth} = useLogin()
+    const {classes} = useStyles()
+
+    if(isAuth){
+        return <Navigate to="/" />
+    }
+    return (
+        <PreloaderExt isLoaded={!!isLoading}>
+            <Flex className={classes.container}>
+                <Paper radius="md" p="xl" withBorder className={classes.wrapper}>
+                    <GoogleOAuthProvider
+                        clientId={'161049581039-40i0002ne5afm5uojndu52g73qoe5g5e.apps.googleusercontent.com'}>
```

Comment:
best practise: all third-party configs data should move to separate files(or config files)